Gruntfile.js

```diff
@@ -165,6 +165,21 @@ module.exports = function (grunt) {
                 src: 'src/main/webapp/js/min/script.js',
                 dest: 'src/main/webapp/js/min/script.clean.js'
             }
+        },
+        protractor : {
+            options : {
+                keepAlive : true,
+                noColor : false,
```

Comment:
To switch on colors we should remove --no-color flags in pom file and grunt file. So maybe it's work for another PR?